dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -427,11 +427,8 @@ public void testOutputDirectory() {
     Truth.assertWithMessage(
             "The dashboard should be created at target/com.google.cloud/libraries-bom/1.0.0")
         .that((Iterable<Path>) outputDirectory)
-        .containsAtLeast(
-            Paths.get("target"),
-            Paths.get("com.google.cloud"),
-            Paths.get("libraries-bom"),
-            Paths.get("1.0.0"))
+        .containsAtLeastElementsIn(
```

Comment:
do we care about order here? This test suggests we don't.

Reply:
Yes, we care about order. The `inOrder()` method 2 lines below verifies the order.

If order is wrong, then we get:
```
required elements were all found, but order was wrong
expected order for required elements: [com.google.cloud, libraries-bom, target, index.html]
but was                             : target/com.google.cloud/libraries-bom/index.html

	at com.google.cloud.tools.opensource.dashboard.FreemarkerTest.testVersionIndex(FreemarkerTest.java:129)
```

Reply:
does this allow for intervening elements or not? API docs are unclear: https://github.com/google/truth/issues/594

Reply:
This allows intervening elements.

```
  @Test
  public void testIntervene() {
    // This passes
    Truth.assertThat(ImmutableList.of("a", "z", "b", "c"))
        .containsAtLeastElementsIn(ImmutableList.of("a", "b", "c")).inOrder();
  }

```

Reply:
Updated to containsExactlyElementsIn, which does not allow intervening elements.

Reply:
We could do that straight-forwardly. It's just Truth that doesn't support that.